e2e/generate_test.go

```diff
@@ -28,3 +28,49 @@ func TestGenerate_FullNode_GoerliNotSupported(t *testing.T) {
 	// Run test case
 	e2eTest.run()
 }
+
+func TestGenerate_FullNode_Lido_Sepolia_NoMEV(t *testing.T) {
+	// Test context
+	var (
+		runErr error
+	)
+	// Build test case
+	e2eTest := newE2ETestCase(
+		t,
+		// Arrange
+		nil,
+		// Act
+		func(t *testing.T, binaryPath string) {
+			runErr = runSedge(t, binaryPath, "generate", "full-node", "--lido", "--network", "sepolia", "--no-mev-boost")
+		},
+		// Assert
+		func(t *testing.T) {
+			assert.NoError(t, runErr, "generate command should succeed with the given arguments")
+		},
+	)
+	// Run test case
+	e2eTest.run()
+}
+
+func TestGenerate_FullNode_Lido_Sepolia(t *testing.T) {
+	// Test context
+	var (
+		runErr error
+	)
+	// Build test case
+	e2eTest := newE2ETestCase(
+		t,
+		// Arrange
+		nil,
+		// Act
+		func(t *testing.T, binaryPath string) {
+			runErr = runSedge(t, binaryPath, "generate", "full-node", "--lido", "--network", "sepolia")
+		},
+		// Assert
+		func(t *testing.T) {
+			assert.Error(t, runErr, "generate command should fail")
```

Comment:
Can we try to catch a more specific error? What if the command fails because something else not related to your intentional misconfiguration?

Reply:
I can't seem to get a more specific error other than ``exit status 1`` , but will look more into it.

Reply:
There should be an error log in the Logs, and if there isn't then we must add it.